Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "EventTimer", "type": "float32" },
+        { "name": "EventList2", "type": "array", "subtype": "object_reference" },
+        { "name": "EventNotifier2", "type": "ObjEventSetterSpawner::EventNotifier2" },
+        { "name": "EventTimer2", "type": "float32" },
+        { "name": "EventList3", "type": "array", "subtype": "object_reference" },
+        { "name": "EventNotifier3", "type": "ObjEventSetterSpawner::EventNotifier3" },
+        { "name": "EventTimer3", "type": "float32" }
+      ]
+    },
+    "ObjHintRingSpawner": {
+      "fields": [
+        { "name": "HintID", "type": "uint32" },
+        { "name": "IsMessageBoot", "type": "bool" },
+        { "name": "OnlyOneBoot", "type": "bool" }
+      ]
+    },
+    "ObjHintRingMultiSpawner": {
+      "fields": [
+        { "name": "HintID", "type": "uint32" },
+        { "name": "HintID2", "type": "uint32" },
+        { "name": "HintID3", "type": "uint32" },
+        { "name": "IsMessageBoot", "type": "bool" },
+        { "name": "OnlyOneBoot", "type": "bool" }
+      ]
+    },
+    "ObjGrouperSpawner": {
+      "fields": [
+        { "name": "TargetList", "type": "array", "subtype": "object_reference" },
+        { "name": "Connect", "type": "bool" }
+      ]
+    },
+    "ObjLayerBoardSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "OnLayer0", "type": "uint8" },
+        { "name": "OnLayer1", "type": "uint8" },
+        { "name": "OffLayer0", "type": "uint8" },
+        { "name": "OffLayer1", "type": "uint8" },
+        { "name": "IsBackEffective", "type": "bool" }
+      ]
+    },
+    "ObjPointMarkerSpawner": {
+      "fields": [
+        { "name": "Width", "type": "float32" },
+        { "name": "Height", "type": "float32" },
+        { "name": "StageType", "type": "ObjPointMarkerSpawner::StageType" }
+      ]
+    },
+    "ObjShadowSpawner": {
+      "fields": [
+        { "name": "Width", "type": "float32" },
+        { "name": "Height", "type": "float32" },
+        { "name": "Depth", "type": "float32" }
+      ]
+    },
+    "TestObjSpawner": {
+      "fields": [
+      ]
+    },
+    "CCTTestObjSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjAppearThornBallSpawner": {
+      "fields": [
+        { "name": "Time", "type": "float32" },
+        { "name": "Distance", "type": "float32" }
+      ]
+    },
+    "ObjChainSpikeBallSpawner": {
+      "fields": [
+        { "name": "Radius", "type": "float32" },
+        { "name": "AngularSpeed", "type": "float32" },
+        { "name": "Phase", "type": "float32" },
+        { "name": "RotateType", "type": "ObjChainSpikeBallSpawner::RotateType" }
+      ]
+    },
+    "ObjChaserBombSpawner": {
+      "fields": [
+        { "name": "ThresholdStubmle", "type": "float32" },
+        { "name": "IsBigChaserMode", "type": "bool" }
+      ]
+    },
```

Comment:
Unused object, makes an appearance in the `stg901` Test Level left over in Sonic Colors: Ultimate. Parameters named based on the `eBigChaserBomb` object in Sonic Unleashed. Set data parser function (0x800D1908, NTSC-J Wii) used for additional help.